.github/workflows/ci.yml

```diff
@@ -86,6 +86,9 @@ jobs:
       - name: Configure minikube
         run: |
           sudo apt install -y conntrack
+          sudo curl -Lo /tmp/minikube https://github.com/kubernetes/minikube/releases/download/v1.16.0/minikube-linux-amd64
+          sudo chmod +x /tmp/minikube
+          sudo mv /tmp/minikube $(which minikube)
```

Comment:
Please switch to using https://github.com/percona-platform/actions-setup-minikube and adding minikube version to `strategy.matrix`

Reply:
(if that's easy to do in this PR)

Reply:
I will do this in separate PR.